src/test/java/guitests/guihandles/MainGuiHandle.java

```diff
@@ -49,4 +59,25 @@ public void use_APP_RESIZE_HOTKEY() {
         guiRobot.sleep(1000);
     }
 
+    public FxRobot sleepForGracePeriod() {
+        return guiRobot.sleep((ModelManager.GRACE_PERIOD_DURATION + 1), TimeUnit.SECONDS);
+    }
+
+    public FxRobot sleep(long duration, TimeUnit timeunit) {
+        return guiRobot.sleep(duration, timeunit);
+    }
```

Comment:
See if you can put the sleep method in the GuiTestBase class and call the MainGuiHandle from there. That way, your code will read `sleep()` instead of `mainGui.sleep()`